students/JeroenSweerts/Lesson4/dict_lab.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#########################Dictionaries 1#########################################
+#Create a dictionary containing “name”, “city”, and “cake” for “Chris” from
+#“Seattle” who likes “Chocolate” (so the keys should be: “name”, etc,
+#and values: “Chris”, etc.)
+d = {}
+d['name'] = 'Chris'
+d['city'] = 'Seattle'
+d['cake'] = 'Chocolate'
+#Display the dictionary.
+print(d)
+#Delete the entry for “cake”.
+del d['cake']
+#Display the dictionary.
+print(d)
+#Add an entry for “fruit” with “Mango” and display the dictionary.
+d['fruit'] = 'Mango'
+print(d)
+#Display the dictionary keys.
+print(d.keys())
+#Display the dictionary values.
+print(d.values())
+#Display whether or not “cake” is a key in the dictionary (i.e. False) (now).
+print('cake' in d.keys())
+#Display whether or not “Mango” is a value in the dictionary (i.e. True).
+print('Mango' in d.values())
+
+#########################Dictionaries 2#########################################
+"""Using the dictionary from item 1: Make a dictionary using the same keys
+but with the number of ‘t’s in each value as the value (consider upper and lower case?)."""
+d2 = {}
+for value in d.values():
```

Comment:
You can use `for key, value in d.items()` to access keys and values in your loop so that your dictionary uses the keys per the instructions